.github/workflows/e2e-wpuf.yml

```
@@ -0,0 +1,122 @@
+        with:
+          report_paths: "tests/e2e/playwright-report/results.xml"
+          include_passed: true,
+          # check_retries: true,
+          # detailed_summary: true,
+      
+      #######--- Remove Later ---#######
+
+      
+
+      - name: Archive playwright artifacts (screenshots, HTML snapshots, videos).
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)